go/kbfs/libkbfs/init.go

```diff
@@ -793,6 +793,10 @@ func doInit(
 	if err != nil {
 		return nil, fmt.Errorf("problem creating service: %s", err)
 	}
+	if registry := config.MetricsRegistry(); registry != nil {
+		service = NewKeybaseServiceMeasured(service, registry)
+	}
+	config.SetKeybaseService(service)
```

Comment:
Was there a reason that service was set later than this? Seems weird. Anyway this is the fix for the kbpki panic.

Reply:
Look at the comment above where this code used to be.  The disk limiter needs to be set first.  All this code is in a very specific order based other past panics, and I expect we'll have problems if you try to change it.

Reply:
Oh, I missed that. But that doesn't make sense -- even before this PR, the service was already created by then. It just wasn't set in config. Is "setting in config" what actually matters here?

Reply:
No, it's the creation that matters, because it registers itself as an RPC service and can start receiving RPCs from the service immediately, some of which depend on the disk limiter.

Reply:
But it didn't. The service is created above at line 791. Even before this move, the disk limiter was enabled after the service creation. Perhaps that disk limiter fix wasn't correct in the first place?

Reply:
Oh I guess that was the original intent, but looks like we changed those RPC handlers since then so that it doesn't matter anymore.  I have no idea, this init sequence is too complicated for me to reason about anymore.  I guess if it works a few times in a real built binary, it's ok for now?

Reply:
I couldn’t even repro the original panic, so this is just a speculative fix. What do you think if we just add a wait logic in the various getters in config? Like config.KBFSOps()

Reply:
I guess we can just leave it like this in the PR, but maybe delete that comment that talks about the disk limiter?  I can't think of a good reason not to set it in the config first.

I'd rather have panics than deadlocks when possible, so I'm not a big fan of putting general loops in the config getters.  I'd rather just have them commented in specific places where they're absolutely necessary.